c7n/filters/core.py

```diff
@@ -335,6 +335,11 @@ def process_value_type(self, sentinel, value):
             # comparisons is intuitively wrong.
             return value, sentinel
 
+        elif self.vtype == 'cidr_size':
+            for i in value:
+                v = parse_cidrs(i)
+            return sentinel, v
+
```

Comment:
Still not 100% sure on this... This is just returning the last value of v, but if I make this a list, it will break the comparisons.